src/mlx/warnings/warnings_checker.py

```diff
@@ -124,6 +115,24 @@ def check(self, content):
         '''
         return
 
+    def initiate_logger(self):
```

Comment:
initialize_loggers

Reply:
Resolved in 8e4828a165a64e5e97704fb07538053f1a5c3553